### ISSUE TITLE
Gravel (558260)

### ISSUE THREAD

**Issue by jolty1** [ValveSoftware/Proton#841]
Game doesn't launch. Fatal Error Message displays

[System Info](https://gist.github.com/jolty1/caaf96b24d0e1e6cff4fb8014a2d3c59)

[steam-558260.log](https://github.com/ValveSoftware/Proton/files/2328998/steam-558260.log)

**Comment by whizse** [comment]
566.815:0024:0025:fixme:d3dcompiler:compile_shader Compilation target "fx_5_0" not yet supported

Testing the demo, it needs a native D3DCompiler_47.dll (64-bit) dropped in steamapps/common/Gravel Demo/gravel/Binaries/Win64

**Comment by mimattr** [comment]
> Testing the demo, it needs a native D3DCompiler_47.dll (64-bit) dropped in steamapps/common/Gravel Demo/gravel/Binaries/Win64

This is correct due to the usage of [trueSKY](https://simul.co/truesky/), more information can be found here: #2136 

Hope that helps.

**Comment by ProjectSynchro** [comment]
This game has issues during FMV cutscenes while using Proton Experimental
Example (ignore vertical letterboxing, I am using a 32:9 monitor):
![image](https://github.com/ValveSoftware/Proton/assets/5131712/0604ec1a-1c62-462a-8636-0cb20d2b5ba1)

Otherwise this game works correctly with no tweaks. I think this is due to some media foundation issues seen in some other unreal engine 4 games.

**Comment by ClaudeLib** [comment]
The FMV issue displayed above still happens when running the game via Proton Experimental and 9.0 Beta. Out of curiosity I tried using the "Movies" folder from the Linux depot 558262 (the MP4 files in it differ in size from the Windows depot 558261 so I assumed they use different encoding), it did not help.

I found it interesting that the intro videos displaying Milestone and Unreal Engine logos play fine, so I tried swapping them with one of the videos that plays later in the game and suffers from the issue. As a result, the "new" intro video, showcasing the career overview, now plays at the start of the game with zero issues, while the Milestone logo video now plays at the start of the career and suffers from the issue depicted above.

Other than that, the Windows version runs perfectly fine. Once the video issue is dealt with, I strongly suggest making the Windows version default, like with Absolute Drift (320140), because the Linux version does not recognize that I own all its DLCs and acts like I don't have any and instead sends me to the store page if I try to access any of the DLC content.

EDIT. Of course, only after writing this I decided to run this game via Proton 8.0-5, and previously bugged videos now play perfectly fine!

**Comment by ClaudeLib** [comment]
Felt like getting back into the game on my OLEDeck and noticed that Proton Experimental fixed the video playback issue.

Can you maybe consider giving it a Verified or at least Playable rating now? Just use the Windows build. The game sets the resolution to 1280x800 and the graphics preset to Ultra on first launch and performs rather well, roughly around 40 FPS, but no lower than 30. When pressing A on the input fields, the keyboard pops up automatically. The UI may be considered tiny on Deck, though.
